dev-libs/mini-xml/mini-xml-2.12.ebuild

```diff
@@ -0,0 +1,57 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit autotools
+
+MY_PN="mxml"
```

Comment:
I'm a bit confused. If upstream's using 'mxml' everywhere, why are we naming it 'mini-xml'?

Reply:
@gentoo/qa, your opinion? Should we continue calling it 'mini-xml' or rename to 'mxml'?

Reply:
@mgorny other distros are using mxml, do not know why we are different

Reply:
I've added an commit, for renaming from `dev-libs/mini-xml` to `dev-libs/mxml` and all reverse deps.

Reply:
Can someone explain me the failed CI?